pkg/volume/glusterfs/glusterfs.go

```diff
@@ -490,13 +495,15 @@ func (r *glusterfsVolumeProvisioner) Provision() (*api.PersistentVolume, error)
 	if len(pv.Spec.AccessModes) == 0 {
 		pv.Spec.AccessModes = r.plugin.GetAccessModes()
 	}
+	sGid := strconv.FormatInt(reqGid, 10)
+	pv.Annotations = map[string]string{"pv.beta.kubernetes.io/gid": sGid}
```

Comment:
I am not sure on, what is @kubernetes/sig-storage 's plan about `pv.beta.kubernetes.io/gid` in 1.5 timeframe. If its available in this release, we could make use of it. @rootfs @childsb @jsafrane  @erinboyd @wongma7 can you please share your thoughts ?